Jenkinsfile.release

```diff
@@ -594,6 +603,11 @@ node('zowe-jenkins-agent-dind-wdc') {
           "docker manifest push ompzowe/server-bundle:latest\n" +
           "docker manifest create ompzowe/server-bundle:v${params.ZOWE_RELEASE_VERSION} ${manifestAmends}\n" +
           "docker manifest push ompzowe/server-bundle:v${params.ZOWE_RELEASE_VERSION}"
+        sh "export DOCKER_CONTENT_TRUST=1\n" +
+          "printf \"${DECRYPTCODE}\n\" | docker trust sign ompzowe/server-bundle:amd64\n" +
+          "printf \"${DECRYPTCODE}\n\" | docker trust sign ompzowe/server-bundle:s390x\n" +
+          "printf \"${DECRYPTCODE}\n\" | docker trust sign ompzowe/server-bundle:latest\n" +
+          "printf \"${DECRYPTCODE}\n\" | docker trust sign ompzowe/server-bundle:v${params.ZOWE_RELEASE_VERSION}"
```

Comment:
I dont know if you need to sign meta tags or if they are signed due to the signing of the original amd64 and s390x images. I will try to see the difference and update the ticket.